### PR TITLE
Added README information about running docs locally together with the troubleshooting guide

### PR DESCRIPTION
**Pull request by oskardudycz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,31 @@ Next Gen docos using VuePress.
 
 ## Local development
 
+### Prerequisites
+
+1. `NodeJs` installed (see [installation guide](https://nodejs.org/en/download//)).
```
Comment (alexeyzimarev) [inline README.md]: NodeJS

File: README.md [hunk 2]
```diff
@@ -4,7 +4,31 @@ Next Gen docos using VuePress.
...
+2. `Yarn` installed (see [installation guide](https://classic.yarnpkg.com/en/docs/install/)).
+3. For Windows development `Windows Subsystem for Linux` (it's neeeded because internally [sed](https://www.gnu.org/software/sed/manual/sed.html) is used in build scripts)\
```
Comment (alexeyzimarev) [inline README.md]: This can now be removed

File: README.md [hunk 3]
```diff
@@ -4,7 +4,31 @@ Next Gen docos using VuePress.
...
+   - [WSL 2 installation guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+   - [Windows Terminal](https://www.microsoft.com/pl-pl/p/windows-terminal/9n0dx20hk701?rtc=1&activetab=pivot:overviewtab)
+
+### Running docs locally
+
 1. Clone the repo
 2. Run `yarn install`
 3. Run `yarn docs:dev`
 
+### Getting external samples
```
Comment (alexeyzimarev) [inline README.md]: I don't think we need this as we plan to change the workflow

**Review by alexeyzimarev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (oskardudycz) [reply README.md]: Removed in https://github.com/EventStore/documentation/pull/163/commits/b48cca4b907699b1cb5bb21347f39fd0ed4855b3.

File: README.md (hunk 3 above)
Comment (oskardudycz) [reply README.md]: Removed in https://github.com/EventStore/documentation/pull/163/commits/b48cca4b907699b1cb5bb21347f39fd0ed4855b3.

File: README.md (hunk 1 above)
Comment (oskardudycz) [reply README.md]: Removed in https://github.com/EventStore/documentation/pull/163/commits/b48cca4b907699b1cb5bb21347f39fd0ed4855b3.

**Comment by oskardudycz** [comment]
@alexeyzimarev I applied your comments. You can re-review the PR.

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**